plugins/plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/api/crypt/server/EncryptTextServiceRegistry.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.crypt.server;
+
+public interface EncryptTextServiceRegistry {
```

Comment:
I'm adding the javadoc. 

About the "legal origin" I guess that you want to be sure that the whole was written by us and does not contain some copy/paste code. I asked Mickael (who created this encryption code) and he confirm that the whole code is from him. Not copy/paste. So no legal issue.
About copyright on images they were already double checked when we added them originally.